### PR TITLE
Added high-level overview of what the existing detailed instructions do

### PR DESCRIPTION
**Pull request by twhateley**

### CODE REVIEW

File: CLOUDFRONT.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+and then how to implement the CloudFront CDN (Content Delivery Network) to securely sit infront of the 
+S3 Bucket, so that content delivery maybe off-loaded from the moodle servers to the CDN.  This will 
+typically result in faster access for users to content due do caching by the CDN, and less load on the
+moodle servers.
+
+The following steps implement the following high level objectives:
+1. Grant the Cloudfront Distribution access to the S3 bucket for ObjectFS
+ 
+  - existing steps in document are for "Legacy access identies"
+		"Use a CloufFront origin access identity (OAI) to access the S3 Bucket"
```
Comment (matthewhilton) [inline CLOUDFRONT.md]: Tiny spelling error in Cloudfront

File: CLOUDFRONT.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+
+  - use "Origin access control settings (recommended)"
```
Comment (matthewhilton) [inline CLOUDFRONT.md:14]: This is a little bit confusing. Are you intending on adding the instructions for this new method?

File: CLOUDFRONT.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+    This configuration has been tested, and also works. This blog post outlines the advantages of the newer option:
+    https://aws.amazon.com/blogs/networking-and-content-delivery/amazon-cloudfront-introduces-origin-access-control-oac/
+
+  - Update S3 Bucket Policy (this step is required for either legacy OAI or Origin access control.
+	"Policy must allow access to CloudFront IAM service principle role". (policy auto generated)
+
+
+2. Restrict viewer access (CloudFront Distribution) to signed requests (Trusted Key Groups)
+
+	- Generate key pair
```
Comment (matthewhilton) [inline CLOUDFRONT.md]: Could you make these links to the corresponding headings? see https://stackoverflow.com/a/51226139

File: CLOUDFRONT.md (hunk 2 above)
Comment (twhateley) [reply CLOUDFRONT.md:14]: I have noted that text in quotes is from the AWS console screen.

There are is much required in terms of instructions, I have added notes under steps 4 & 5 of "Create CloudFront Distribution" where this option is implemented.

File: CLOUDFRONT.md (hunk 3 above)
Comment (twhateley) [reply CLOUDFRONT.md]: done

**Pull request merged**